tests/std/tests/GH_003572_fstream_seekp_0_cur/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\usual_latest_matrix.lst

```

Comment:
It appears that this test could be run down to C++17 (with `string_view`). We use the "latest" matrix only when C++23 features are necessary.